include/ebpf_api.h

```diff
@@ -148,7 +143,7 @@ extern "C"
      * @param[out] value_size Size of values in the eBPF map.
      * @param[out] max_entries Maximum number of entries in the map.
      */
```

Comment:
same here (but ok to do in a future PR)

Reply:
I can fix this in the next PR -- I am going to work on implementing the APIs next.